worker-languagedetection-container/testcases/automated/CAF_4001/irish.yaml.testcase

```diff
@@ -51,35 +51,35 @@ expectedOutputData:
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "99"
+        - data: "58"
```

Comment:
Significant change here, cannot identify irish anymore. If it is not supported then remove this testcase and make sure that irish is not referred to as a supported language.